packages/runtime/src/utils/isElementInViewport.ts

```diff
@@ -0,0 +1,14 @@
+export function isElementInViewport(element: HTMLElement, offset = 0): boolean {
```

Comment:
Consider renaming `offset`
```suggestion
export function isElementInViewport(element: HTMLElement, margin = 0): boolean {
```
```suggestion
export function isElementInViewport(element: HTMLElement, threshold = 0): boolean {
```